src/main/java/duke/Event.java

```diff
@@ -0,0 +1,20 @@
+package duke;
+
+public class Event extends Task{
+    protected String at;
+
+    public Event(String description, int taskNumber, String at) {
+        super(description, taskNumber);
+        this.at = at;
+    }
+
```

Comment:
I feel the code's readability can be benefited more with an '@Override' comment for overridden methods, even though it is only optional.